### PR TITLE
Changed single quotes to double quotes in code snippets where double quotes trigger the "copy" icon, making it easier to copy commands.

### PR DESCRIPTION
**Pull request by Abhinavcode13**
@Sybrand & @Ideonvn  , PTAL.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,11 +90,11 @@ For many Harmony projects, the free tier of Mapbox will be sufficient. Refer to
 
 When you run a script or the web server, select a configuration by setting the `ZEN_ENV` environmental variable. This environmental variable maps directly to folder names in `config/`, and will cause the `config` module to export the contents of that particular configuration.
 
-      Say there is configuration directory named `usa`. We can specify that configuration with the following:
+Say there is configuration directory named `usa`. We can specify that configuration with the following:
 
-      ```bash
-      export ZEN_ENV='usa'
-      ```
+```
```
Comment (Sybrand) [inline README.md:95]: Why have you removed the bash hint?

File: README.md [hunk 2]
```diff
@@ -137,31 +137,86 @@ All instructions going forward will assume that the environment variables have b
...
 Run `make dev-prepare-database`
+```
 
 ### Run Web Server
 
-1. Run `make up`
-2. In a separate terminal, connect to the running web container, run `SERVICE=web make exec-bash`
-3. Now run: `source venv/bin/activate && ./scripts/create_user.py --username=me@mydomain.com --password=password --first_name=admin --last_name=istrator --site_admin`
+1. Run
```
Comment (Sybrand) [inline README.md:146]: We intentionally made us if single quotes here as it result in an easier to read instruction set. Using triple quotes throws it to the next line, which doesn't read as easily.

**Review by Sybrand — COMMENTED** [review]
Could you please rationalise why you feel that these changes improve the readability or utility of the instructions?

File: README.md (hunk 2 above)
Comment (Abhinavcode13) [reply README.md:146]: I agree, but I feel this way is more clear.

**Comment by Abhinavcode13** [comment]
> Could you please rationalise why you feel that these changes improve the readability or utility of the instructions?

As it makes it more clear to a contributor, these are the commands that are to be executed in sequence instead of using
`single slash`.

I would suggest using single backquote markup only for the following things:

- commands to be typed in, or parts of commands to be typed in
- file names or command names

File: README.md [hunk 3]
```diff
@@ -137,31 +137,86 @@ All instructions going forward will assume that the environment variables have b
 
 ### Prepare Database
 
+```
```
Comment (Sybrand) [inline README.md:140]: This markdown is incorrect, it's making Run be part of the markdown. I think you meant to say
Run:
```
make dev-prepare-database
```

**Review by Sybrand — COMMENTED** [review]
I think I understand what you mean re. "streamlined command copying" - are you saying that single quotes down bring up the "copy" icon in github, but double quotes do - making it easier to copy commands for use in the command line using the copy function built into the browser?

If that's the case, can you please update your pull request to describe the intention of the PR in more detail, and then I'll take another look.

**Comment by Abhinavcode13** [comment]
> I think I understand what you mean re. "streamlined command copying" - are you saying that single quotes don't bring up the "copy" icon in github, but double quotes do - making it easier to copy commands for use in the command line using the copy function built into the browser?
> 
> If that's the case, can you please update your pull request to describe the intention of the PR in more detail, and then I'll take another look.

Yes, that is what I meant.

**Comment by Abhinavcode13** [comment]
Updated it , PTAL.

File: README.md (hunk 3 above)
Comment (Abhinavcode13) [reply README.md]: Resolved

File: README.md (hunk 1 above)
Comment (Abhinavcode13) [reply README.md]: Resolved

File: README.md [hunk 4]
```diff
@@ -137,31 +137,86 @@ All instructions going forward will assume that the environment variables have b
...
+
+  ```
+  make ps
+  ```
+- It's useful to have a terminal open on the web instance:
+  
+  ```
+  SERVICE=web make exec-bash
+  ```
+  furthermore running the below command will activate the python virtual environment.
```
Comment (Sybrand) [inline README.md:209]:
```suggestion
  furthermore running the command below will activate the python virtual environment:
```

File: README.md [hunk 5]
```diff
@@ -137,31 +137,86 @@ All instructions going forward will assume that the environment variables have b
 
 ### Prepare Database
 
-Run `make dev-prepare-database`
+   ```
```
Comment (Sybrand) [inline README.md:140]: Run:
```suggestion
```bash
```

File: README.md [hunk 6]
```diff
@@ -137,31 +137,86 @@ All instructions going forward will assume that the environment variables have b
...
+   Run `make dev-prepare-database`
```
Comment (Sybrand) [inline README.md:141]: That isn't a valid command - you need to remove the single quotes, and "Run" isn't a command.
```suggestion
make dev-prepare-database
```

File: README.md [hunk 7]
```diff
@@ -137,31 +137,86 @@ All instructions going forward will assume that the environment variables have b
...
+   ```
```
Comment (Sybrand) [inline README.md:142]:
```suggestion
```
```

**Review by Sybrand — CHANGES_REQUESTED** [review]
I've made some specific comments, but in general, could you please always specify language when creating a code snipped.
e.g.
\```bash
as opposed to
\```

**Comment by Abhinavcode13** [comment]
> I've made some specific comments, but in general, could you please always specify language when creating a code snipped. e.g. ```bash as opposed to ```

Yes - I agree that would be more appropriate.

**Comment by Abhinavcode13** [comment]
Can you review this now @Sybrand ?

**Comment by Abhinavcode13** [comment]
PTAL now, @Sybrand

**Pull request closed without merging**